distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -0,0 +1,33 @@
+priority "grant";
+
+// Double forward slashes "//" indicate comments and have no effect on policies.
+
+
+//                        Content Directory Monitor (CDM) Permissions
+// Adding required permissions for a new Content Directory Monitor.
+//
+// If the directory being monitored is located in DDFs home directory, then adding these permissions
+// is not required and these instructions can be ignored.
+//
+// Configuring CDM requires adding read and write permissions to the directory being monitored. For
+// each configured CDM, the following 2 permissions are required:
+//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read, write";
+// Replacing <DIRECTORY_PATH> with the path of the directory being monitored.
+//
+// Line 1 gives the CDM the permissions to read from the monitored directory path. Line 2 gives the
+// CDM the permissions to recursively read and write from the monitored directory path, specified
+// by the directory path's suffix "${/}-".
+//
+// Adding new permissions requires a system restart to take affect.
```

Comment:
✏️ If this file is meant to be used for any configuration related policies, not just CDM, we may want to move some of those comments (restart, trailing slashes) at the top since they would apply to everything.